duckduckgo/Android
# Updated translations

Task/Issue URL: https://app.asana.com/0/414730916066338/1202248365771000/f

### Description

Updated translations in all languages for downloads and automatic fireproof projects

### Steps to test this PR

- Install from this branch
- Change device language to any (but English)
- [ ]  Check downloads section is translated
- [ ] Check Settings > Fireproof Sites screen is translated

### No UI changes
